org.eclipse.scanning.event/src/org/eclipse/scanning/event/ConsumerImpl.java

```diff
@@ -190,46 +191,39 @@ protected void terminate(KillBean kbean) {
 			stop();
 			if (kbean.isDisconnect()) disconnect();
 		} catch (EventException e) {
-			logger.error("An internal error occurred trying to terminate the consumer "+getName()+" "+getConsumerId());
+			LOGGER.error("An internal error occurred trying to terminate the consumer "+getName()+" "+getConsumerId());
 		}
 		if (kbean.isExitProcess()) {
 			try {
 				Thread.sleep(2500);
 			} catch (InterruptedException e) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either re-interrupt this method or rethrow the "InterruptedException". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2142)